pkg/proxy/ipvs/netlink_linux.go

```diff
@@ -123,7 +123,7 @@ func (h *netlinkHandle) ListBindAddress(devName string) ([]string, error) {
 }
 
 // GetLocalAddresses lists all LOCAL type IP addresses from host based on filter device.
-// If filter device is not specified, it's equivalent to exec:
+// If dev is not specified, it's equivalent to exec:
 // $ ip route show table local type local proto kernel
```

Comment:
IIUC, the way of fetching LOCAL node IP has changed - we no longer use the `ip route show ...` command, right? So, we should change the comment as well.

Reply:
Actually, we still using `h.RouteListFiltered` to show routes, which equivalent to exec `ip route show...`, what changed is the way we filter routes that we don't want.

Reply:
But I think this comment can be simplify.

Reply:
Oh, I think I get it. But...we can keep optimizing this piece code without using ip route show, e.g. 

```
link, err := h.LinkByName(dev)
addrs := link.Attrs()
```

We can improve it in another PR.